### ISSUE TITLE
Gas Fees Calculator (Multi Currency, Network & Txn Types)

### ISSUE THREAD

**Issue by feliche93** [ethereum/ethereum-org-website#5099]
Before submitting this suggestion, be sure to read our [listing policy for dev tools](https://ethereum.org/en/contributing/adding-developer-tools/)

**Product name:**

Gas Fees Calculator (Multi Currency, Network & Txn Types)

**Product description:** 

Calculate gas fees in your local currency for different transaction types on Mainnet, Arbitrum, Binance Smart Chain, Avalanche, Polygon, Fantom and Harmony. 

Launched on December 5, 2021: https://twitter.com/FelixVemmer/status/1467532744519565323

**Where do you think it should go?**

https://ethereum.org/en/developers/docs/gas/#strategies-for-you-to-reduce-gas-costs

**GitHub URL**

I am currently in the part of checking my codebase for secrets, but will soon open-source it. It's mostly using the zapper.fi API though: https://docs.zapper.fi/zapper-api/api-getting-started

**Docs URL**

https://www.cryptoneur.xyz/gas-fees-calculator

**Is the product open source?**

Not yet, currently reviewing my personal website repo for secrets and then planning to make it all open source.

**Additional context**

You probably ask how is this different from https://l2fees.info/. I took a lot of inspiration and ideas from this project but wanted to make it even more useful the following way:

- *Support for int. currencies*: Most tools are always in USD which make it hard for ppl to understand how prices are in their local currencies. That is why I added multi-currency support
- *Multi Chain*: Just like L2 Fees I wanna support the etherum strategy for ppl to move to L2s and understand how usable and low cost the usage is. This is why I added a total of 7 networks
- *Multi Transaction Types*: For me, the most important feature was to dynamically adjust the gas used parameter. This allows people to get an estimated cost for almost any transaction, given that they can adjust this parameter. I am working on adding more "standard" use cases such as gnosis safe creation that helps ppl to for example more safely store their funds

**Comment by feliche93** [comment]
Hey @minimalsm I saw you added the label `needs grooming`. I was wondering if there's anything needed from my side on this or if you're missing some infos :)

**Comment by minimalsm** [comment]
Hey @feliche93, thanks for raising this. Just wanted to double-check some things on our end. Happy to get this added but won't have the capacity to do it myself anytime soon. Can you open a PR to add it?

**Comment by feliche93** [comment]
@minimalsm I checked out the how-to contribute section, forked the repo and created a new branch `gas_fees_calculator` and pushed the changes to my forked repo:

https://github.com/feliche93/ethereum-org-website/commit/d78e1579398a1ff3d9b76b256895f34dbbb61c02

Any more steps I need to do? How would I open a PR on your site now? 

Thanks!

**Comment by feliche93** [comment]
Hi @minimalsm, I wanted to check-in and ask if there's anything else that you'd need from my side :)

**Comment by minimalsm** [comment]
Yeah, please open a PR into this repo. Thanks 🙏

**Comment by feliche93** [comment]
@minimalsm thanks for the heads up, just opened the PR 🙏

https://github.com/ethereum/ethereum-org-website/pull/5622

**Comment by oceanemil** [comment]
Can you please add calculations for typical Safe operations like transfer from Safe with 2 signers or owners?

**Comment by feliche93** [comment]
@oceanemil yes happy to do it. Do you know how much gas such a txn consumes?

If you need any other txns just post the name and the gas used and I'll add them all 🙂
